wsd/ClientRequestDispatcher.cpp

```diff
@@ -1284,9 +1285,12 @@ bool ClientRequestDispatcher::handleWopiAccessCheckRequest(const Poco::Net::HTTP
 
             if (!probeSession->getSslVerifyMessage().empty())
             {
-                status = CheckStatus::CertificateValidation;
-
-                LOG_DBG("Result ssl: " << probeSession->getSslVerifyMessage());
+                if (probeSession->getSslVerifyMessage() == "self-signed certificate") {
```

Comment:
this problem here is that this checks the result of `X509_verify_cert_error_string()` which is "human readable".

1. are we sure the text is never localized
2. are we sure it never changes?

Otherwise we'd better use the result of `SSL_get_verify_result(_ssl)`

Reply:
The API would need to expose properly errors code indeed, it would be better.
This `getSslVerifyMessage` is currently used only for logging, here I have a different use-case.

We can expect it to be always english (https://linux.die.net/man/3/x509_verify_cert_error_string) and its string to be stable, as they even are documented.

Reply:
I switched to use the ssl result.